src/main/java/ca/bc/gov/chefs/etl/forms/pcd/hrRecords/model/HRRecordsPractitioner.java

```diff
@@ -0,0 +1,237 @@
+    public void setFteEquivalent(String fteEquivalent) {
+        this.fteEquivalent = fteEquivalent;
+    }
+    public String getPaymentModality() {
+        return paymentModality;
+    }
+    public void setPaymentModality(String paymentModality) {
+        this.paymentModality = paymentModality;
+    }
+    public String getDateHired() {
+        return dateHired;
+    }
+    public void setDateHired(String dateHired) {
+        this.dateHired = dateHired;
+    }
+    public String getFiscalYear() {
+        return fiscalYear;
+    }
+    public void setFiscalYear(String fiscalYear) {
+        this.fiscalYear = fiscalYear;
+    }
+    public String getPeriod() {
+        return period;
+    }
+    public void setPeriod(String period) {
+        this.period = period;
+    }
+    public String getEmploymentStatus() {
+        return employmentStatus;
+    }
+    public void setEmploymentStatus(String employmentStatus) {
+        this.employmentStatus = employmentStatus;
+    }
+    public String getDateEmploymentStatusChanged() {
+        return dateEmploymentStatusChanged;
+    }
+    public void setDateEmploymentStatusChanged(String dateEmploymentStatusChanged) {
+        this.dateEmploymentStatusChanged = dateEmploymentStatusChanged;
+    }
+    public String getRecordCreationDate() {
+        return recordCreationDate;
+    }
+    public void setRecordCreationDate(String recordCreationDate) {
+        this.recordCreationDate = recordCreationDate;
+    }
+    public String getNotes() {
+        return CSVUtil.replaceCarriageReturnLineFeed(notes);
+    }
+    public void setNotes(String notes) {
+        this.notes = notes;
+    }
+    public String getLegacyWebformId() {
+        return legacyWebformId;
+    }
+    public void setLegacyWebformId(String legacyWebformId) {
+        this.legacyWebformId = legacyWebformId;
+    }
+    
+    @Override
+    public String getFileName() {
+        return null;
+    }
+    @Override
+    public List<String> getCsvElements() {
+        List<String> elements = new ArrayList<String>();
+        elements.add(submissionId);
+        elements.add(hrRecordId);
+        elements.add(practitionerType);
+        elements.add(practitionerName);
+        elements.add(practitionerFirstName);
+        elements.add(practitionerLastName);
+        elements.add(practitionerRole);
+        elements.add(practitionerBillingNumber);
+        elements.add(practitionerBillingNumberNotAvailable);
+        elements.add(specialty);
+        elements.add(otherSpecialty);
+        elements.add(groupRole);
+        elements.add(additionalGroupDetails);
+        elements.add(duration);
+        elements.add(fteEquivalent);
+        elements.add(paymentModality);
+        elements.add(dateHired);
+        elements.add(fiscalYear);
+        elements.add(period);
+        elements.add(employmentStatus);
+        elements.add(dateEmploymentStatusChanged);
+        elements.add(recordCreationDate);
+        elements.add(notes);
```

Comment:
I missed one thing here. For notes and any other "textarea" fields you need to actually call the getter in getCsvElements so that the exported version is correct.